web/src/app/alerts/CreateAlertSpeedDial.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react'
+import { KeyChange as ServicesIcon } from 'mdi-material-ui'
+import { Notifications as AlertsIcon } from '@material-ui/icons/'
+import { makeStyles } from '@material-ui/styles'
+import classnames from 'classnames'
+import AlertForm from './components/AlertForm'
+import CreateAlertDialog from './CreateAlertDialog'
+import SpeedDial from '../util/SpeedDial'
+
+const useStyles = makeStyles(theme => ({
+  speedDial: {
+    position: 'fixed',
+    bottom: '2em',
+    right: '2em',
+    zIndex: 9001,
+  },
+}))
+
+export default props => {
+  const classes = useStyles()
+
+  const [showCreateAlertForm, setShowCreateAlertForm] = useState(false)
+  const [showCreateAlertDialog, setShowCreateAlertDialog] = useState(false)
+
+  const showForm = form => {
```

Comment:
I don't think we need this, we can just call the proper method from the relevant action below.

e.g. instead of calling `showForm('createAlertByLabel')` let's just call `setShowCreateAlertForm(true)`.